src/theory/arith/kinds

```diff
@@ -8,7 +8,7 @@ theory THEORY_ARITH ::CVC5::theory::arith::TheoryArith "theory/arith/theory_arit
 typechecker "theory/arith/theory_arith_type_rules.h"
 
 properties stable-infinite
-properties check propagate ppStaticLearn presolve notifyRestart
+properties propagate ppStaticLearn presolve notifyRestart
```

Comment:
```suggestion
properties ppStaticLearn presolve notifyRestart
```

Reply:
`propagate()` is defined in `theory_arith.h`, though?

Reply:
Oh I see, it looked as though you were deleting the propagate property, I forgot this is not 100% standardized yet.

Nevermind :)